### PR TITLE
#158495467 Setup scss linting

### PR DESCRIPTION
**Pull request by williamolojede**
#### What Does This PR Do?
This PR setup stylelint for scss files in the client folder.

#### Description Of Task To Be Completed
- Add stylelint configuration
- Add npm scripts to automate running linting

#### Any Background Context You Want To Provide?
There was no way to enforce a uniform format of writing scss and also ensure following of best practices when writing scss. 

#### How can this be manually tested?
- Pull this branch to your local
- Install all dependencies with yarn
- Run `yarn run style-lint` to run lint on all scss file

#### What are the relevant pivotal tracker stories?
[#158495467](https://www.pivotaltracker.com/story/show/158495467)

#### What are the relevant GitHub issues?
N/A

#### Screenshot(s)
N/A

### CODE REVIEW

**Comment by nwochaadim** [comment]
@adesege or @ipheghe any of you can test that this works as it ought to and if yes, LGTM! on my end. @williamolojede before merging this, sync with me on the format of the final commit message

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (nwochaadim) [inline .stylelintrc.json]: I believe this file should be in the client folder. The backend guys should not care about this

Comment (williamolojede) [reply .stylelintrc.json]: The sylelint cli command has an option to pass in the path of the config file, fixing this won't be a problem will do it right away.

**Comment by nwochaadim** [comment]
@adesege thanks for the suggestion. This can be something we can look into later in a separate PR, but for now, lets allow this PR go in.

**Pull request merged**